doc/user/content/sql/create-source/postgres.md

```diff
@@ -0,0 +1,166 @@
+
+Before creating the source in Materialize, you must:
+
+1. Set up your Postgres database to allow logical replication.
+2. Ensure that the user for your Materialize connection has `REPLICATION` privileges.
+3. Create a Postgres [publication](https://www.postgresql.org/docs/10/logical-replication-publication.html), or replication data set, containing the tables to be streamed to Materialize. Since Postgres sources are materialized (kept in memory) in their entirety, we strongly recommend that you limit publications only to the data you need to query.
+
+Once you create a materialized source from the publication, the source will contain the raw data stream of replication updates. You can then break the stream out into views that represent the publication's original tables with [`CREATE VIEWS`](/sql/create-views/). You can treat these tables as you would any other source and create other views or materialized views from them.
+
+#### Postgres schemas
+
+`CREATE VIEWS` will attempt to create each upstream table in the same schema as Postgres. For example, if the publication contains tables` "public"."foo"` and `"otherschema"."foo"`, `CREATE VIEWS` is the equivalent of
+
+```
+CREATE VIEW "public"."foo";
+CREATE VIEW "otherschema"."foo";
+```
+
+Therefore, in order for `CREATE VIEWS` to succeed, all upstream schemas included in the publication must exist in Materialize as well, or you must explicitly specify the downstream schemas and rename the resulting tables. For example:
+
+```sql
+CREATE VIEWS FROM "mz_source"
+("public"."foo" AS "foo", "otherschema"."foo" AS "foo2");
+```
+#### Postgres replication slots
+
+If you stop or delete Materialize without first dropping the Postgres source, the Postgres replication slot isn't deleted and will continue to accumulate data. In such cases, you should manually delete the Materialize replication slot to recover memory and avoid degraded performance in the upstream database. Materialize replication slot names always begin with `materialize_` for easy identification.
+
+#### Restrictions on Postgres sources
+
+- Materialize does not support changes to schemas for existing publications. You will need to drop the existing sources and then recreate them after creating new publications for the updated schemas.
+- Sources can only be created from publications that use [data types](/sql/types/) supported by Materialize. Attempts to create sources from publications which contain unsupported data types will fail with an error.
+- Materialize only supports [TOASTED](https://www.postgresql.org/docs/current/storage-toast.html) values for append-only tables. Practically speaking, you can include rows with TOASTED values as long as they are never updated or deleted, or you can disable TOAST on the original Postgres table. If a TOASTED column is updated, the source will enter an errored state that renders all per-table views inaccessible.
```

Comment:
This might be my bad english, but I initially interpreted this sentence as "if you have an append-only table, it must *only* contain TOASTED values". Whereas the meaning we want is "if you have an append-only table, it may contain TOASTED values"

Reply:
Good catch! Amended to "Materialize does not support [TOASTED](https://www.postgresql.org/docs/current/storage-toast.html) values except in append-only tables."